protocol/messenger_communities.go

```diff
@@ -47,6 +49,19 @@ var updateActiveMembersInterval = 24 * time.Hour
 
 const discordTimestampLayout = "2006-01-02T15:04:05+00:00"
 
+var importRateLimiter = rate.NewLimiter(rate.Every(importSlowRate), 1)
+
+const (
+	importSlowRate          = time.Second / 1
+	importFastRate          = time.Second / 100
+	importMessagesChunkSize = 10
+)
+
+const (
+	maxChunkSizeMessages = 1000
+	maxChunkSizeBytes    = 1500000
```

Comment:
This one doesn't seem to be used any longer..

Reply:
Actually, both are no longer used here

Reply:
It is still used when importing the community in `RequestImportDiscordCommunity` which is a bit different than importing messages from archives.

Reply:
Ah sorry, forgot